setup.cfg

```diff
@@ -10,4 +10,4 @@ filterwarnings=
 # Ignore node_modules and cloned repos when not in a virtual environment
 exclude = node_modules/*,tests/*,src/*
 max-line-length = 160
-ignore = C815,C816,W503,E203
+ignore = C815,C816,W503,E203,E704
```

Comment:
Out of scope for this PR, but it may be useful to have inline comments/links on why we're ignoring some of these rules?

Reply:
Ticket raised to look into it 👍